assignments/objects.js

```diff
@@ -15,31 +15,89 @@ let example = {
   "name": "Example",
   "email": "examples@you.edu",
   "gender": "F"
-}
+};
 
 // Write your intern objects here:
 
+let internOne = {
+    "id": 1,
+    "email": "mmelloy0@psu.edu",
+    "name": "Mitzi",
+    "gender": "F"
+};
+
+let internTwo = {
+    "id": 2,
+    "email": "kdiben1@tinypic.com",
+    "name": "Kennan",
+    "gender": "M"
+};
+
+let internThree = {
+    "id": 3,
+    "email": "kmummery2@wikimedia.org",
+    "name": "Keven",
+    "gender": "M"
+};
+
+let internFour = {
+    "id": 4,
+    "email": "gmartinson3@illinois.edu",
+    "name": "Gannie",
+    "gender": "M"
+};
+
+let internFive = {
+    "id": 5,
+    "email": "adaine5@samsung.com",
+    "name": "Antonietta",
+    "gender": "F"
+};
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(internOne.name);
+
 // Kennan's ID
 
+console.log(internTwo.id);
+
 // Keven's email
 
+console.log(internThree.email);
+
 // Gannie's name
 
+console.log(internFour.name);
+
 // Antonietta's Gender
 
+console.log(internFive.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+internTwo.speak = function() {
```

Comment:
Nice work! You can also add a method directly to the object. We'll talk more about methods when we get into Classes later this week :smile: